src/components/List/Row.js

```diff
@@ -28,15 +28,19 @@ const ListRow = ({ children, rowItem, index, onOverflowClick, ...props }) => (
 
 ListRow.defaultProps = {
   children: null,
-  onExpandShow: "subTitle"
+  onExpandShow: "subTitle",
+  onExpandItem: ListRowContent.defaultProps.onExpandItem,
+  onCollapseItem: ListRowContent.defaultProps.onCollapseItem,
```

Comment:
Can you please explicitly pass the `onExpandItem` and `onCollapseItem` props from the `ListRow` to `ListRowContent`? Also, please add details to the catalog markdown file containing the components' documentation.